KeepingYouAwake/Preferences/KYABatteryPreferencesViewController/zh-Hant.lproj/KYABatteryPreferencesViewController.strings

```diff
@@ -0,0 +1,26 @@
+/* Class = "NSTextFieldCell"; title = "Battery Level"; ObjectID = "pvY-d7-nKX"; */
+"pvY-d7-nKX.title" = "電池電量";
+
+/* Class = "NSButtonCell"; title = "Deactivate when battery capacity is below"; ObjectID = "gx4-rK-1he"; */
+"gx4-rK-1he.title" = "低於以下電量時清空杯子";
```

Comment:
I like the humor of the `清空杯子`. However, I'd like to suggest we can use `低於以下電量時關閉`.

Reply:
I'm not entirely sure which is better, here's why I've chosen these terms:

For "open & close the app", we use

- `launch` => `開啟`
- `quit`  => `結束`

For "activate & deactivate the feature", we use

- `activate` and `starts preventing sleep` => `斟滿`
- `deactivate` and `allowing sleep` => `清空杯子`

Previously I used `啟用`/`停用` for `activate`/`deactivate`, but users could be confused if it is together with `開啟` (e.g., `開啟時啟用`).

---

Besides, the term `停用` could be regarded as "disable the feature", but it will only deactivate (empty the cup) once only per the condition is met.

![image](https://user-images.githubusercontent.com/2075566/222963438-7bb4fd34-baa8-480b-abf8-0db708937526.png)

![image](https://user-images.githubusercontent.com/2075566/222963454-011d343f-e112-4d65-bec6-e5f47f0dff36.png)